src/pages/hegel/guides/science-of-logic-introduction.mdx

```diff
@@ -0,0 +1,184 @@
+
+Hegel notes a further error in the case where one is discouraged by a contradiction in the understanding and resolves to find accord and stability in sensuous reality. But, and here Hegel criticizes the Kantian transcendental standpoint once more, if one resolves to look to sensuous reality as the criterion of truth, in the same instance one cannot determine the thing in question to have _any_ independent truth since the truth of the matter is settled _only_ as it appears. "This is like attributing right insight to someone," Hegel writes, "with the stipulation, however, that he is not fit to see what is true but only what is false" (Hegel, 2010, 26/21.30). To put this another way, if by truth one understands a independent standard, then this immediately clashes with the knowledge through appearances, since knowing here is totally dependent on appearances and cannot make _any_ independent claims about the matter at hand. To emphasize this point further, if the transcendental arrangement is actually granted, then one cannot truly speak of laws of nature or even state that the sky is there the moment one turns one's gaze towards the ground. This is why, if Hegel is right, this claim that sensuous reality is the sole arbiter of truth leads to absurdity.
+
+Finally, Hegel then points out that as the understanding (according to transcendental idealism) cannot apply its determinations or concepts to things in themselves, it must therefore mean that these determinations are in themselves something untrue and consequently cannot be turned upon itself or anything else related to pure reason. The standard set for the understanding is based on experience and empirical objects, but the categories that define these empirical objects are not themselves empirical&mdash;they are precisely pure&mdash;and so the categories of the understanding cannot apply to anything other than sensuous reality. But this is incoherent, since the categories themselves are pure, their validity cannot come from sensuous reality. "If they cannot be determinations of _the thing in itself_, still less can they be determinations of the _understanding_, to which one ought to concede at least the dignity of a thing" (Hegel, 2010, 27/21.31). In this Kantian regime, according to Hegel, one cannot consistently use the tools by which the understanding navigates the world to also navigate the realm of ideas and thoughts, with the result being that the understanding (or the mind) cannot be a thing, much less be said to exist.
```

Comment:
It's clear that the main target in this section is Kant or TI. Perhaps it would be pedagogically helpful to mention that at the start.

Reply:
Yep, added!